### PR TITLE
Update hosts.yml

### PR DESCRIPTION
**Pull request by Levure**
Added GalacSYS shared hosting (https://www.galacsys.net/hebergement_mutualise_details.html)

### CODE REVIEW

File: _data/hosts.yml [hunk 1]
```diff
@@ -548,6 +548,22 @@
             semver: 7.1.1    
     last_scanned_at: '2017-01-09T15:57:19+0000'
 -
+    name: 'GalacSYS'
+    url: 'https://www.galacsys.net/hebergement_mutualise_details.html'
+    type: shared
+    default: null
+    versions:
+        44:
```
Comment (philsturgeon) [inline _data/hosts.yml]: I really hope you meant PHP 5.4?

Comment (Levure) [inline _data/hosts.yml]: It's not a typo, that's really PHP4.4 (see http://php4info.galacsys.com/).
I assume this is intended for legacy websites made by Mathusalem himself 👴🏼 ?

Comment (philsturgeon) [inline _data/hosts.yml]: Good god. 

Well, 4.4 is so incredibly old, this website doesn't care to track such things. Can you just omit it from the PR, and we can pretend this didn't happen? 🙈

Comment (Levure) [inline _data/hosts.yml]: Done ! ;-)

**Pull request merged**